### PR TITLE
Feature/cjs sizes

### PR DESCRIPTION
**Pull request by erykpiast**
It resolves these two issues:
 * https://github.com/nanoutils/nanoutils/issues/212 (latest `size-limit` provides it out-of-the-box)
 * https://github.com/nanoutils/nanoutils/issues/211

### CODE REVIEW

**Comment by erykpiast** [comment]
@Beraliv do you think you will find a time this week to look at this? :)

**Comment by Beraliv** [comment]
> @Beraliv do you think you will find a time this week to look at this? :)

Sure, will find time later this evening! ⏰

Thank you for you help in advance! ♥️

File: misc/cli/size.js [hunk 1]
```diff
@@ -63,144 +66,158 @@ const getStatisticsRows = methods => {
...
-      formatName('total'),
-      formatSize(total.sizeNoRamda),
-      formatSize(total.ramdaSize),
-      formatDiff(total.sizeNoRamda, total.ramdaSize)
+  
+  printTable(methodsWithSizes, longestName)
+
+  if (!args._.length) {
+    await Promise.all([
+      writeSizesFile('esmSize', 'SIZES.md', methodsWithSizes),
```
Comment (Beraliv) [inline misc/cli/size.js:198]: 🔥

File: package.json [hunk 2]
```diff
@@ -55,7 +56,7 @@
     "np": "^3.0.4",
     "ramda": "^0.25.0",
     "rimraf": "^2.6.2",
-    "size-limit": "^0.13.2",
+    "size-limit": "4.5.0",
```
Comment (Beraliv) [inline package.json:59]: Thank you for that in advance! 🔥

**Pull request merged**